examples/golang/docker-compose-go.yaml

```
@@ -0,0 +1,18 @@
+version: "3.8"
+
+services:
+  go_plugin_compile:
+    build:
+      context: ../shared/build
+    command: >
+      bash -c "
+      bazel build  //examples/golang/simple:simple.so
+      && cp -a ./bazel-bin/examples/golang/simple/simple.so /output"
+    environment:
+    - BUILD_UID=${UID:-1000}
+    - TEST_TMPDIR=${ENVOY_TEST_TMPDIR:-/tmp}
```

```suggestion
    - TEST_TMPDIR=/tmp
```

i think there is no point making this configurable as it is hard coded just below